testing/gearmand/APKBUILD

```diff
@@ -0,0 +1,75 @@
+# Contributor: Aaron Hurt <ahurt@ena.com>
+# Maintainer: Aaron Hurt <ahurt@ena.com>
+pkgname=gearmand
+pkgver="1.1.12"
+pkgrel=2
+pkgdesc="Gearman job server and client"
+url="http://gearman.org"
+arch="all"
+license="BSD"
+depends="libgearman"
+depends_dev="boost-dev gperf libevent-dev util-linux-dev mariadb-dev
+	hiredis-dev openssl-dev sqlite-dev libmemcached-dev"
+makedepends="$depends_dev"
+install="$pkgname.pre-install $pkgname.post-deinstall"
+subpackages="$pkgname-doc libgearman libgearman-dev"
```

Comment:
I’ve discussed this PR on IRC and we agreed that it’ll be better to name these subpackages `gearmand-libs` (*.so libs) and `gearmand-dev` (header files etc.) It’s more common naming schema in Alpine. That said, it may look odd in this case, because of trailing “d” in gearmand that usually means “daemon.” Naming it “gearman” would make more sense, but Gearman’s developers decided to name it “gearmand” for whatever reason and we prefer to name packages the same as the packaged software is officially named. WDYT?